src/get_opening_hours.ts

```diff
@@ -38,6 +46,62 @@ function toHourGroup(hourRange: HourRange): HourGroup | null {
   return { from: fromWithLeadingZero, to: toWithLeadingZero };
 }
 
+// Pattern to match absolute date like "Jan 26" or "Apr 13"
+const absoluteDatePattern = /^([A-Z][a-z]{2})\s+(\d+)$/;
+
+// Check if a day group string starts with an absolute date pattern
+function startsWithAbsoluteDate(dayGroup: string): boolean {
+  return /^[A-Z][a-z]{2}\s+\d/.test(dayGroup);
+}
+
+function parseAbsoluteDates(dateRangesStr: string): AbsoluteDate[] {
+  const absoluteDates: AbsoluteDate[] = [];
+  const parts = dateRangesStr.split(/,/);
+
+  for (const part of parts) {
+    const trimmedPart = part.trim();
+    const match = trimmedPart.match(absoluteDatePattern);
+    if (match?.[1] && match[2]) {
```

Comment:
Destructure array immediately instead of assignments below this line. Use early throw instead of if else.